src/js/components/Tabs.js

```diff
@@ -52,6 +70,17 @@ export default class Tabs extends Component {
       activeTitle: activeTitle
     });
 
+    var tabsContentPad;
```

Comment:
I think any padding should be controlled by the caller. I've discussed with Allan and he's planning to merge the div and Box and have no padding. Which would make Brand Central easier. :)